azext_edge/edge/util/file_operations.py

```diff
@@ -0,0 +1,109 @@
+# coding=utf-8
+# ----------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License file in the project root for license information.
+# ----------------------------------------------------------------------------------------------
+
+import csv
+import json
+import yaml
+import os
+from pathlib import PurePath
+from typing import List, Optional
+from azure.cli.core.azclierror import FileOperationError
+from knack.log import get_logger
+
+logger = get_logger(__name__)
+
+
+def convert_file_content_to_json(file_path: str):
```

Comment:
What matters more is the contents of the file rather than the extension for processing.